pkg/localConfig/localConfig.go

```diff
@@ -5,69 +5,120 @@ import (
 	"os/user"
 	"path/filepath"
 
+	"github.com/datreeio/datree/pkg/cliClient"
 	"github.com/lithammer/shortuuid"
 	"github.com/spf13/viper"
 )
 
-type ConfigContent struct {
-	CliId         string
+type LocalConfig struct {
+	Token         string
+	ClientId      string
 	SchemaVersion string
 }
 
-type LocalConfig struct {
+type TokenClient interface {
+	CreateToken() (*cliClient.CreateTokenResponse, error)
+}
+
+type LocalConfigClient struct {
```

Comment:
I think it should be called `LocalConfig` and not `LocalConfigClient`. this struct contains a client